modules/ply/src/ply-loader.js

```diff
@@ -14,8 +14,7 @@ const DEFAULT_OPTIONS = {
 export default {
   name: 'PLY',
   extension: 'ply',
-  format: 'text',
-  parseBinary: parsePLY,
-  parseText: parsePLY,
+  parseTextSync: parsePLY,
+  parseSync: parsePLY,
```

Comment:
We have separate parsers for e.g. `parseBinaryGLTF` and `parseTextGLTF`. 
Shall we split `parsePLY` to `parseBinaryPLY` and `parseTextPLY` or at least add a comment that both formats are handled inside `parsePLY`?

Reply:
Done